<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogProduct</class>
 <widget class="QDialog" name="DialogProduct">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Información de Producto</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_22">
   <item>
    <widget class="QLabel" name="label_17">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Información Principal</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_17">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Producto</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leCodigo"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Tipo de Prodcuto</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leTipoProd"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Envase</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leEnvase"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Código Alternativo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leCodigoAlt"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Marca</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leMarca"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_18">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Costos</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_21">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Costo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leCosto">
            <property name="inputMethodHints">
             <set>Qt::ImhFormattedNumbersOnly</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Costo Público</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leCostoP">
              <property name="inputMethodHints">
               <set>Qt::ImhFormattedNumbersOnly</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_19">
            <item>
             <widget class="QLabel" name="label_21">
              <property name="text">
               <string>Formula Para Calcular Costo Publico. (Usar x como valor del costo)   </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leFormulaCP"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pbCalcularCP">
          <property name="text">
           <string>Calcular</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Costo Mayorista</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leCostoM">
              <property name="inputMethodHints">
               <set>Qt::ImhFormattedNumbersOnly</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_20">
            <item>
             <widget class="QLabel" name="label_22">
              <property name="text">
               <string>Formula Para Calcular Costo Mayorista (Usar x como valor del costo)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leFormulaCM"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pbCalcularCM">
          <property name="text">
           <string>Calcular</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_19">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Stock</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Cantidad Deposito</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leCantidadAtras">
          <property name="inputMethodHints">
           <set>Qt::ImhDigitsOnly</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Min Stock Frente</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leMinStockFrente">
          <property name="inputMethodHints">
           <set>Qt::ImhDigitsOnly</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Min Stock Deposito</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leMinStockDep">
          <property name="inputMethodHints">
           <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Cantidad Frente</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leCantidadFrente">
          <property name="inputMethodHints">
           <set>Qt::ImhDigitsOnly</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_20">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Información Expandida</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_18">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Descripción</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="pteDesc"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_16">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Ubicación</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leUbicacion"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_15">
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Columna</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leColumna">
              <property name="inputMethodHints">
               <set>Qt::ImhDigitsOnly</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_14">
            <item>
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Estante</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leEstante">
              <property name="inputMethodHints">
               <set>Qt::ImhDigitsOnly</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="pbAceptar">
       <property name="text">
        <string>Aceptar</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="leError">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#aa0000;&quot;&gt;El codigo no puede estar vacio&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
